### PR TITLE
sandesh scons changes to complie under src/contrail-common

### PR DESCRIPTION
**Pull request by Jackjvs**
Partial-Bug: #1729402

### CODE REVIEW

**Review by anishmehta — APPROVED** [review]

**Review by sundaresanr — APPROVED** [review]

File: sandesh/compiler/SConscript [hunk 1]
```diff
@@ -19,7 +19,7 @@ for flag in rmflags:
 
 env['CCFLAGS'] = cflags
 
-env.Append(CPPPATH = ['#tools/sandesh/compiler','#build/include'])
+env.Append(CPPPATH = ['#src/contrail-common/sandesh/compiler','#build/include'])
```
Comment (klash) [inline sandesh/compiler/SConscript:22]: The `#` is top-of-sandbox... so I think this needs to be `#controller/src/contrail-common`

File: sandesh/library/c/test/SConscript [hunk 2]
```diff
@@ -12,7 +12,7 @@ env = SandeshEnv.Clone();
 env.Append(CPPPATH = [
         Dir('#build/include').abspath,
         Dir('#controller/src').abspath,
-        Dir('#tools').abspath,
+        Dir('#src/contrail-common').abspath,
```
Comment (klash) [inline sandesh/library/c/test/SConscript:15]: ditto

File: sandesh/library/common/test/SConscript [hunk 3]
```diff
@@ -11,7 +11,7 @@ env = SandeshEnv.Clone()
 env.Append(CPPPATH = [
         Dir('#build/include').abspath,
         Dir('#controller/src').abspath,
-        Dir('#tools').abspath,
+        Dir('#src/contrail-common').abspath,
```
Comment (klash) [inline sandesh/library/common/test/SConscript:14]: ditto

File: sandesh/library/cpp/SConscript [hunk 4]
```diff
@@ -13,7 +13,7 @@ env.Append(CPPPATH = [
     '#build/include',
     '#controller/src',
     Dir('.').abspath,
-    '#tools',
+    '#src/contrail-common',
```
Comment (klash) [inline sandesh/library/cpp/SConscript:16]: ditto

File: sandesh/library/cpp/SConscript [hunk 5]
```diff
@@ -66,20 +66,20 @@ def WebServerBuild(wenv, abs_pth, loc, inp):
...
-('tools/sandesh/library/common/webs','css/images/sort_desc.png'),
-('tools/sandesh/library/common/webs','css/images/sort_desc_disabled.png'),
-('tools/sandesh/library/common/webs','css/style.css'),
-('tools/sandesh/library/common/webs','js/bootstrap.min.js'),
-('tools/sandesh/library/common/webs','js/DT_bootstrap.js'),
-('tools/sandesh/library/common/webs','js/jquery-2.0.3.min.js'),
-('tools/sandesh/library/common/webs','js/jquery.dataTables.min.js'),
-('tools/sandesh/library/common/webs','js/util.js'),
-('tools/sandesh/library/common/webs','universal_parse.xsl')]
+('src/contrail-common/sandesh/library/common/webs','css/bootstrap.min.css'),
```
Comment (klash) [inline sandesh/library/cpp/SConscript:69]: all these too

File: sandesh/compiler/SConscript (hunk 1 above)
Comment (Jackjvs) [reply sandesh/compiler/SConscript:22]: # is top of sand box. controller, vrouter and src are siblings. So it should be src/controller-common (that is how it is currently)

File: sandesh/library/c/test/SConscript (hunk 2 above)
Comment (Jackjvs) [reply sandesh/library/c/test/SConscript:15]: same as above

File: sandesh/library/common/test/SConscript (hunk 3 above)
Comment (Jackjvs) [reply sandesh/library/common/test/SConscript:14]: same as above

File: sandesh/library/cpp/SConscript (hunk 4 above)
Comment (Jackjvs) [reply sandesh/library/cpp/SConscript:16]: same as above

File: sandesh/library/cpp/SConscript (hunk 5 above)
Comment (Jackjvs) [reply sandesh/library/cpp/SConscript:69]: same as above

**Pull request merged**